addon/components/ember-popper-base.js

```diff
@@ -122,6 +122,16 @@ export default class EmberPopperBase extends Component {
   }
 
   /**
+   * ================= CALLBACKS =================
+   */
+
+  onCreate() {
```

Comment:
Does popper.js allow us to make these `null` instead of empty functions? I'd like to avoid creating unused functions/calling no-op functions if possible.

Reply:
I'd like to go with option 2 where we omit the functions in the options hash if they are not used.

I'm going to put a PR into Popper.js to guard against non-function `onCreate` and `onUpdate` args while removing their internal no-ops. No reason for both libs to take a memory and CPU hit running non-existent functions.